src/main/java/kg/jarkyn/server/Incoming/Requester.java

```diff
@@ -0,0 +1,9 @@
+package kg.jarkyn.server.Incoming;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public interface Requester {
```

Comment:
I had a comment from Uku that ClientSocket vs ServerSideSocket are not necessarily true descriptions, so I went for something very generic. I definitely need an interface to help with testing, but still haven't finalised name here, will give it another thought